common/mixins.py

```diff
@@ -1,3 +1,86 @@
+from auditlog.context import set_actor
+from auditlog.signals import accessed
+from rest_framework import mixins
+
+
+class LogListAccessMixin(mixins.ListModelMixin):
```

Comment:
Looks generic enough to be perhaps be worthy to include in django-auditlog-extra itself, or what do you think?

Reply:
See https://github.com/City-of-Helsinki/palvelutarjotin/pull/384#discussion_r2026770812